docs/openapi/lpa-codes-openapi-v1.yml

```diff
@@ -52,12 +53,26 @@ paths:
             schema:
               type: object
               properties:
-                lpa:
-                  type: string
-                  # not sure what 'LPA' refers to here, see ticket IN-71
-                actor:
-                  type: string
-                  # not sure what 'actor' refers to here, see ticket IN-71
+                lpas:
+                  type: array
+                  items:
+                    type: object
+                    properties:
+                      lpa:
+                        type: string
+                      actor:
+                        type: string
+                      dob:
+                        type: string
+                        format: date
+              example:
+                 lpas:
+                   - lpa: "eed4f597-fd87-4536-99d0-895778824861"
+                     actor: "12ad81a9-f89d-4804-99f5-7c0c8669ac9b"
```

Comment:
Do we know the source of these UUIDs? Sirius stores UIDs for LPAs and actors, but they're 12-digit numbers rather than formatted as UUIDs.

Reply:
It doesn't matter what they are really. I just used UUIDs for my examples because I like them.